examples/basic/example_09_tracemapping.py

```diff
@@ -0,0 +1,277 @@
+
+    selection_manager = ExtAPI.SelectionManager
+    selection = ExtAPI.SelectionManager.CreateSelectionInfo(
+        SelectionTypeEnum.GeometryEntities
+    )
+    selection.Ids = list_of_body_ids
+    selection_manager.NewSelection(selection)
+
+    model = ExtAPI.DataModel.Project.Model
+    named_sel = model.AddNamedSelection()
+    named_sel.Name = ns_name
+    named_sel.Location = selection
+    selection_manager.ClearSelection()
+
+
+create_named_selection_from_id_list("board_layers", board_bodyids)
+create_named_selection_from_id_list("components", component_bodyids)
+
+#make a selection to be used with mesh methods
+selection_manager = ExtAPI.SelectionManager
+selection = ExtAPI.SelectionManager.CreateSelectionInfo(
+    SelectionTypeEnum.GeometryEntities
+)
+selection.Ids = board_bodyids
+selection_manager.NewSelection(selection)
+
+mesh = ExtAPI.DataModel.Project.Model.Mesh
+
+mesh_method = mesh.AddAutomaticMethod()
+mesh_method.Location = selection
+mesh_method.Method = MethodType.Sweep
+mesh_method.ElementOrder = ElementOrder.Linear
+mesh_method.SweepNumberDivisions = 1
+
+mesh_sizing = mesh.AddSizing()
+mesh_sizing.ElementSize = Quantity("0.25 [mm]")
+mesh.GenerateMesh()
+
+
+# Defining External Data Object  for Importing Trace
+external_data_files = Ansys.Mechanical.ExternalData.ExternalDataFileCollection()
+external_data_files.SaveFilesWithProject = True
+external_data_file = Ansys.Mechanical.ExternalData.ExternalDataFile()
+external_data_files.Add(external_data_file)  # Single File
+external_data_file.Identifier = "edb"
+external_data_file.Description = ""
+external_data_file.IsMainFile = False
+external_data_file.FilePath = def_file
+external_data_file.ImportSettings = (
+    Ansys.Mechanical.ExternalData.ImportSettingsFactory.GetSettingsForFormat(
+        Ansys.Mechanical.DataModel.MechanicalEnums.ExternalData.ImportFormat.ECAD
+    )
+)
+import_settings = external_data_file.ImportSettings
+import_settings.UseDummyNetData = False
+imported_trace_group = Model.Materials.AddImportedTraceExternalData()
+imported_trace_group.ImportExternalDataFiles(external_data_files)
+
+allImpTraces = ExtAPI.DataModel.GetObjectsByType(
+    Ansys.Mechanical.DataModel.Enums.DataModelObjectCategory.ImportedTrace
+)
+
+imp_trace = [
+    x for x in allImpTraces if x.Parent.ObjectId == imported_trace_group.ObjectId
+][0]
+imp_trace.Activate()
+imp_trace.InternalObject.GeometryDefineBy = 1
+
+NSall = ExtAPI.DataModel.Project.Model.NamedSelections.GetChildren[
+    Ansys.ACT.Automation.Mechanical.NamedSelection
+](True)
+ns_object = [i for i in NSall if i.Name == "board_layers"][0]
+imp_trace.Location = ns_object
+imp_trace.PropertyByName("PROPID_ExternalData").InternalValue = 1
+
+
+layers = imp_trace.Layers
+num_layers = layers.Count
+for layer in layers:
+    layer["Trace Material"] = "Copper Alloy"
+vias = imp_trace.Vias
+num_vias = vias.Count
+for via in vias:
+    via["Plating Material"] = "Copper Alloy"
+imp_trace.Import()
```

Comment:
@koubaa , Bug fixed with imported load in 241, so keeping it open till release as you recommended